Packs/Core/ReleaseNotes/3_0_96.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Added the argument **download_package** to command **core-get-distribution-url** to download an installation package file.
```

Comment:
please fix to:
Added support for the **download_package** argument in the **xdr-get-distribution-url** command to enable downloading an installation package file.